cmd/access/node_builder/access_node_builder.go

```diff
@@ -2088,42 +2102,24 @@ func (builder *FlowAccessNodeBuilder) Build() (cmd.Node, error) {
 
 			}
 
-			preferredENIdentifiers, err := flow.IdentifierListFromHex(backendConfig.PreferredExecutionNodeIDs)
-			if err != nil {
-				return nil, fmt.Errorf("failed to convert node id string to Flow Identifier for preferred EN map: %w", err)
-			}
-
-			fixedENIdentifiers, err := flow.IdentifierListFromHex(backendConfig.FixedExecutionNodeIDs)
-			if err != nil {
-				return nil, fmt.Errorf("failed to convert node id string to Flow Identifier for fixed EN map: %w", err)
-			}
-
-			builder.ExecNodeIdentitiesProvider = commonrpc.NewExecutionNodeIdentitiesProvider(
```

Comment:
The initialization is moved earlier as a general module. This change is required to fix cohort4 test case `integration/tests/access/cohort4/access_store_tx_error_messages_test.go`. Without this change, the access node will panic because the tx error message store receives a nil ExecNodeIdentitiesProvider